rust/lance-encoding/src/compression_algo/fastlanes.rs

```diff
@@ -0,0 +1,2142 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright The Lance Authors
+
+// NOTICE:
+// This file is a modification of the `fastlanes` crate: https://github.com/spiraldb/fastlanes
+// It is modified to allow a rust stable build
+// The original code can be accessed at https://github.com/spiraldb/fastlanes/blob/8e0ff374f815...
```

Comment:
```suggestion
// The original code can be accessed at 
https://github.com/spiraldb/fastlanes/blob/7b1df5362e54f38af602463fc6785200af6a6f1c/src/bitpacking.rs
```
Sorry, my link was cutoff

Reply:
fixed